pkg/services/librarypanels/librarypanels.go

```diff
@@ -18,7 +18,6 @@ type Service interface {
 	LoadLibraryPanelsForDashboard(c *models.ReqContext, dash *models.Dashboard) error
 	CleanLibraryPanelsForDashboard(dash *models.Dashboard) error
 	ConnectLibraryPanelsForDashboard(c *models.ReqContext, dash *models.Dashboard) error
-	ImportDashboard(c *models.ReqContext, dashboard *simplejson.Json, importedID int64) error
```

Comment:
ah, this is nice, because I thought when this was added that it feels wrong, that this service needed a special case for this scenario.